DiscordBot.DataAccess/IEventsSheetsService.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using DiscordBot.Models;
+
+namespace DiscordBot.DataAccess
+{
+    interface IEventsSheetsService
+    {
+        Task AddEventAsync(string name, string description, DateTime time, Dictionary<string, string> roleEmotes);
```

Comment:
I'd recommend leaving out the roles to start with. You want to start with simple functionality, get that working then extend it 🙂

Reply:
Yeah I agree, @kwunkiuma and I were a bit undecided on the whole roles/signups interface so it makes sense to leave that for now and sort it when the first part of the service is done. I've simplified the interface to just editing the events metadata sheet (the first table above) for the time being, so events can be added, edited and removed but no-one can sign up to them yet. I hope that's ok 🙂